build.gradle

```diff
@@ -552,6 +552,7 @@ jlink {
     addOptions('--strip-debug', '--compress', '2', '--no-header-files', '--no-man-pages')
     launcher {
         name = 'JabRef'
+        jvmArgs = ['-Xmx4g', '-XX:+UseZGC']
```

Comment:
Would need to be configured for Mac on the Pupeline as well

Reply:
Is there a different **`build.gradle`** in for MacOS? I hope NOT.

Note that my first pushes adapted the `deployment*` files. Then I learned that the parameters passed to the application are configured in the Gradle build file.

Reply:
jlink is called manually for mac as part of jpackage

Reply:
Some things, are garbage collected daily in my brain :p

Reply:
Ported: [`6a536e4` (#10334)](https://github.com/JabRef/jabref/pull/10334/commits/6a536e44460ebae3df11115ef8d8eb6e313c0f2c)